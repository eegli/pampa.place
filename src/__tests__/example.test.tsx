import { render } from '@testing-library/react';
import { min } from '@utils';
import Error from '../components/error';

describe('it tests', () => {
  it('works', () => {
    expect(min(2, 3)).toEqual(2);
  });

  it('renders', () => {
    const { container } = render(<Error />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
