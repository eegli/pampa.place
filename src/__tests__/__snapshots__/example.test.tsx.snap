// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it tests renders 1`] = `
<div
  class="MuiBox-root css-1wcaknn"
>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation0 MuiAlert-root MuiAlert-outlinedError MuiAlert-outlined css-134l0rj-MuiPaper-root-MuiAlert-root"
    role="alert"
  >
    <div
      class="MuiAlert-icon css-1ytlwq5-MuiAlert-icon"
    >
      <svg
        aria-hidden="true"
        class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit css-1vooibu-MuiSvgIcon-root"
        data-testid="ErrorOutlineIcon"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M11 15h2v2h-2zm0-8h2v6h-2zm.99-5C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zM12 20c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
        />
      </svg>
    </div>
    <div
      class="MuiAlert-message css-acap47-MuiAlert-message"
    >
      <div
        class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom MuiAlertTitle-root css-yslrua-MuiTypography-root-MuiAlertTitle-root"
      >
        Something went wrong
      </div>
      <p />
      <p>
        <strong>
          Message:
        </strong>
      </p>
      <pre
        style="white-space: pre-wrap; overflow: auto;"
      />
    </div>
  </div>
</div>
`;
